### ISSUE TITLE
[FABRIC] Nostalgia Tweaks seems to be incompatible with a mod/mods from Better Minecraft modpack

### ISSUE THREAD

**Issue by xtosutosu** [Adrenix/Nostalgic-Tweaks#17]
## Version Information
> Please fill out the following information. Replace each `x` with a number:

Minecraft Version: 1.18.2
Fabric Loader Version: 0.14.9
Fabric API Version: 0.58.0
Nostalgic Tweaks Version: 1.0.5

## Describe the Bug
> A clear and concise description of what the bug is (e.g., startup crash, runtime crash, or something not working as intended).

The game console log gets spammed with constant error messages.
Some parts (chunks) of the world stop loading in, they do not refresh and cause the game to become unresponsive.

## Crash Report
> If applicable, please include a link to a complete crash report. You can use Gist (https://gist.github.com) to create this link. These reports can be found in `.minecraft/crash-reports`.

https://mclo.gs/tm0E1px

## Bug Context
> If applicable, add some context to the bug. If a crash occurred while you were in-game, what happened before the crash? For example, "I fell from a high place and before the hurt sound could play the game crashed."

I believe it might be caused by some mods from the Better Minecraft/ Better MC modpack although I am not sure which.
It has to be mods that either do changes to biomes or other terrain features.
**Additional mods I put into my mods folder are: Create and Ad Astra although without them the problem still occurs** 

## Screenshots
> If applicable, add screenshots to help explain your problem.

![obraz](https://user-images.githubusercontent.com/17273281/187213177-014b85b5-11ef-43a6-9f8e-02e5090e8557.png)

**Comment by Adrenix** [comment]
I can't seem to find any code related to Nostalgic Tweaks that is causing issues in the log you submitted. I loaded up the modpack with N.T installed and wasn't able to reproduce those errors. The issue might be related to another mod since the error that was being spammed was a null pointer exception with `biomeEntry` being `null`.

**Comment by Adrenix** [comment]
If you are able to find the mod that is causing a conflict, then please feel free to reopen this issue. :)

**Comment by xtosutosu** [comment]
Hi, I found the reason my game behaved like that. Luckily, **your mod is not the cause**.

Turns out it was not a mod, but any datapacks (or mods that are datapacks under the hood) that modify the terrain and/or biomes (ex. Terralith or Hybrid Beta, the second pains me the most) will end up conflicting with mods that change or add new biomes (ex. Promenade or Oh The Biomes You'll Go). 

Kind of a bummer but it is what it is
